### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by rbrtmrtn**
*Issue #, if available:*

*Description of changes:*

This PR proposes a few changes to improve style, consistency, and readability of the issue template. It also adds a few newlines so Markdown blocks are parsed correctly. Just my suggestions -- admittedly a little opinionated 😊 -- but hopefully this is helpful to others submitting issues. Thanks!

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ searching the issues list. If there is a duplicate, re-open or add a comment to
...
-Github Issues is for bug reports and feature requests. If you have general support
-questions, the following locations are a good place:
+# Need general help?
+
+GitHub issues are for bug reports and feature requests. If you have general support
+questions, the following resources are a good place to start:
 
 - Slack channel (#samdev): https://awssamopensource.splashthat.com/
-- Post a question in StackOverflow  with "aws-sam-cli" tag
+- Post a question on Stack Overflow with the tag `aws-sam-local`
```
Comment (jfuss) [inline .github/ISSUE_TEMPLATE.md]: Is there a reason for switching this back to `aws-sam-local`. This should match the current project name. Can you update this back to `aws-sam-cli`, or explain why we should keep the `*-local`?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -19,18 +20,23 @@ questions, the following locations are a good place:
 <!-- Briefly describe the problem you are facing -->
 
 **Steps to reproduce the issue:**
+
 1.
 2.
 3.
 
+**Expected result:**
```
Comment (jfuss) [inline .github/ISSUE_TEMPLATE.md]: Let's keep previous order of Observed and then Expected.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jfuss) [reply .github/ISSUE_TEMPLATE.md]: Hmm.. Looks like we need to update that on SO, but not sure how to be honest. Let me see if I can't figure that out. If that is a long process, we can address that later. :)

Comment (rbrtmrtn) [reply .github/ISSUE_TEMPLATE.md]: @jfuss Sounds like a plan :) I opened a discussion on StackOverflow Meta to have it renamed -- will keep you posted.

Comment (sanathkr) [reply .github/ISSUE_TEMPLATE.md]: @rbrtmrtn  How did the discussion in StackOverflow go?

Comment (rbrtmrtn) [reply .github/ISSUE_TEMPLATE.md]: I got one upvote... :)

https://meta.stackoverflow.com/questions/368189/retag-aws-sam-local-to-aws-sam-cli

I'm familiar enough with SO admin stuff to know if there's some way to bump the issue. Anyone else?

**Comment by sanathkr** [comment]
Closing this because of inactivity

**Pull request closed without merging**